src/app/embed/adapters/feed.adapter.ts

```diff
@@ -2,8 +2,10 @@ import { EMBED_FEED_URL_PARAM, EMBED_FEED_ID_PARAM, EMBED_EPISODE_GUID_PARAM } f
 import {Injectable} from '@angular/core';
 import { Http, Response } from '@angular/http';
 import { Observable } from 'rxjs/Observable';
-import { AdapterProperties, DataAdapter } from './adapter.properties'
+import { AdapterProperties, DataAdapter } from './adapter.properties';
+let sha1 = require('sha1');
```

Comment:
were you going to vendor that module I sent you? I really want to avoid requires since they don't do well with modern build system optimizations.